sormas-ui/src/main/java/de/symeda/sormas/ui/statistics/AbstractStatisticsView.java

```diff
@@ -42,5 +50,14 @@ public void refreshMenu(SubMenu menu, String params) {
 		if (UserProvider.getCurrent().hasUserRight(UserRight.DATABASE_EXPORT_ACCESS)) {
 			menu.addView(DatabaseExportView.VIEW_NAME, I18nProperties.getCaption(Captions.statisticsDatabaseExport), params);
 		}
+
+		String sormasStatsUrl = FacadeProvider.getConfigFacade().getSormasStatsUrl();
+		if (sormasStatsUrl != null) {
```

Comment:
Use `StringUtils.isNotBlank` instead; safer than only a null check in case someone accidentally typed in a whitespace